Sources/SDFCore/MacroInitializerRewriter.swift

```diff
@@ -142,20 +144,64 @@ private final class MacroInitializerRewriter: SyntaxRewriter {
         let infixOperand = InfixOperatorExprSyntax(foldedNode),
         infixOperand.leftOperand.trimmed.description.starts(with: "self.")
```

Comment:
I think we likely want to be a little more specific here. It's not just that we begin with `self.`. We explicitly only want to group assignments of the form:
```swift
self.ivar = assignedValue
```
if and only if `ivar == assignedValue`